packages/datadog_tracking_http_client/lib/src/tracking_http_client.dart

```diff
@@ -107,6 +107,8 @@ class DatadogTrackingHttpClient implements HttpClient {
           attributes[DatadogRumPlatformAttributeKey.traceID] = traceId;
           attributes[DatadogRumPlatformAttributeKey.spanID] = spanId;
         }
+        attributes[DatadogRumPlatformAttributeKey.rulePsr] =
```

Comment:
As I understand, `tracingSamplingRate` is between 0 and 100, so `rulePsr` should be `rum.tracingSamplingRate / 100`?

Reply:
Yes, good catch!